routes/homestats.py

```diff
@@ -0,0 +1,105 @@
+    daily_subs = db.query(models.User).filter(models.User.status == "subscribed").count()
+    profile_deletions = db.query(models.User).filter(models.User.status == "deleted").count()
+    return {
+        "total_users": total_users,
+        "daily_subscriptions": daily_subs,
+        "profile_deletions": profile_deletions
+    }
+
+# ---- AGENTS ----
+@router.get("/agents")
+def get_agent_stats(db: Session = Depends(get_db)):
+    total_active = db.query(models.Agent).filter(models.Agent.status == "online").count()
+    appointment = db.query(models.Agent).filter(models.Agent.type == "appointment").count()
+    emailing = db.query(models.Agent).filter(models.Agent.type == "emailing").count()
+    telephony = db.query(models.Agent).filter(models.Agent.type == "telephony").count()
+    return {
+        "total_active_agents": total_active,
+        "agent_types": {
+            "appointment": appointment,
+            "emailing": emailing,
+            "telephony": telephony
+        }
+    }
+
+# ---- ACTIVITIES ----
+@router.get("/activities")
+def get_activity_stats(db: Session = Depends(get_db)):
+    messages = db.query(models.Activity).filter(models.Activity.type == "message").count()
+    emails = db.query(models.Activity).filter(models.Activity.type == "email").count()
+    calls = db.query(models.Activity).filter(models.Activity.type == "call").count()
+    return {
+        "daily_messages": messages,
+        "daily_emails": emails,
+        "daily_calls": calls
+    }
+
+# ---- HEALTH ----
+@router.get("/health")
+def get_health_stats(db: Session = Depends(get_db)):
+    latest = db.query(models.Health).order_by(models.Health.created_at.desc()).first()
+    return {
+        "failed_integrations": latest.failed_integrations if latest else 0,
+        "long_response_delays": latest.long_response_delays if latest else 0
+    }
+
+# ---- OVERVIEW (all stats together) ----
+@router.get("/overview")
+def get_overview(db: Session = Depends(get_db)):
+    total_users = db.query(models.User).count()
+    daily_subs = db.query(models.User).filter(models.User.status == "subscribed").count()
+    profile_deletions = db.query(models.User).filter(models.User.status == "deleted").count()
+
+    total_active = db.query(models.Agent).filter(models.Agent.status == "online").count()
+    appointment = db.query(models.Agent).filter(models.Agent.type == "appointment").count()
+    emailing = db.query(models.Agent).filter(models.Agent.type == "emailing").count()
+    telephony = db.query(models.Agent).filter(models.Agent.type == "telephony").count()
+
+    messages = db.query(models.Activity).filter(models.Activity.type == "message").count()
+    emails = db.query(models.Activity).filter(models.Activity.type == "email").count()
+    calls = db.query(models.Activity).filter(models.Activity.type == "call").count()
+
+    latest = db.query(models.Health).order_by(models.Health.created_at.desc()).first()
+
+    return {
+        "users": {
```

Comment:
again when you have created fdifferent api fro the same thing and than a single api for same purpose again